snowpack/src/config.ts

```diff
@@ -68,6 +68,8 @@ const DEFAULT_CONFIG: SnowpackUserConfig = {
     sourceMaps: false,
     watch: false,
     htmlFragments: false,
+    jsxFactory: undefined,
```

Comment:
if undefined, just leave unset in this default object. You'll need to update the SnowpackConfig TypeScript interface as well.

Reply:
I added it because I saw [hmrPort](https://github.com/snowpackjs/snowpack/pull/1838/files/c788d6addd888d73548516d466d9dba7da95b92f#diff-8ffa4446c9ee1a7fc3e0379d80b4665ac70b44dc9c872e915e92a8da57208602L58) was set to undefined above in dev options. Makes sense, I'll remove it.